templates/sag-is-mft/test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh -e
+
+# if managed image
+if [ -d $SAG_HOME/profiles/SPM ] ; then
+    # point to local SPM
+    export CC_SERVER=http://localhost:8092/spm
+    export is_instance_name=${is_instance_name:-default}
+
+    echo "Verifying managed container $CC_SERVER ..."
+    sagcc get inventory products -e MFTSupport --wait-for-cc
+
+    export CC_WAIT=180
+    
+    echo "Verifying fixes ..."
+    sagcc get inventory fixes -e MAT_10.1_Server_Fix
```

Comment:
What is this is tested against 10.3? Is the fix name still MAT_10.1_Server_Fix?
It's better to use internal fix name, e.g. wMFix.xxxx which is cross-release name